packages/core/strapi/lib/services/entity-service/__tests__/entity-service.test.js

```diff
@@ -21,7 +21,11 @@ describe('Entity service', () => {
       'Can decorate',
       async method => {
         const instance = createEntityService({
-          strapi: {},
+          strapi: {
+            eventHub: {
```

Comment:
if you removed the function this shouldn't be necessary ?

Reply:
Removed while you were reviewing :sweat_smile: